slidez-fe-core/src/services/ws/ws-endpoint.ts

```diff
@@ -0,0 +1,13 @@
+const WsEndpoint = Object.freeze({
+    ENDPOINT: process.env.REACT_APP_WEB_SOCKET_ENDPOINT || '',
+    SNAPSHOT_TOPIC: process.env.REACT_APP_WEB_SOCKET_SNAPSHOT_TOPIC || '',
+    CREATED_POLL_TOPIC:
+        process.env.REACT_APP_WEB_SOCKET_CREATED_POLL_TOPIC || '',
+    ANSWERED_POLL_TOPIC:
+        process.env.REACT_APP_WEB_SOCKET_ANSWERED_POLL_TOPIC || '',
+    SNAPSHOT_QUEUE: process.env.REACT_APP_WEB_SOCKET_SNAPSHOT_QUEUE || '',
+    CREATE_POLL_QUEUE: process.env.REACT_APP_WEB_SOCKET_CREATE_POLL_QUEUE || '',
+    ANSWER_POLL_QUEUE: process.env.REACT_APP_WEB_SOCKET_ANSWER_POLL_QUEUE || '',
```

Comment:
If I understand them correctly, write these variables statically, they won't change from server to server